### DISCUSSION TITLE
NCD CN L2 Cohort 1 (2021/09/17 ~ 2021/11/26)

### DISCUSSION THREAD

**Discussion by think-in-universe** [near-x/learn-near-community#33] [Ideas]
## Introduction to NCD L2

Congratulations to everyone who has received your NCD L1 certificate. You're now able to build your DApp and contract with NEAR, even though something basic yet. This a great move towards the next milestone of building something great together. 

NCD L2 is aimed at building something REAL, either by yourself from scratch, or learning by contributing to existing projects, and working together with developers worldwide. 

### TestNet Project v.s. Contributor

We have two tracks in NCD L2 course: TestNet Project Track or Contributor Track.

The [TestNet Project Track](https://learnnear.club/courses/near-certified-developer-level-2-testnet-project-track/) guides you through the basic procedure of Web3 project creation and production: 

- Idea -> Prototype -> Build -> Release -> Feedback -> Improvement

The [Contributor Track](https://learnnear.club/courses/near-certified-developer-level-2-contributor-track/) guides you through the basic procedure of Web3 project open source contribution ([orbit model](https://github.com/orbit-love/orbit-model)):

- Explore -> Participate -> Contribute -> Build -> Team

###  Schedule

Two-Month Journey: 2021/09/17 ~ 2021/11/26

1. L2 Kick-Off: 1st week of each month
2. Weekly Check-in: every Friday
3. Mid-Term Demo: by end of the first month: e.g. 2021/10/29
4. Final Demo: by end of the two months: e.g. 2021/11/26

### Course

1. **From 0 To 1**: for product builders and entrepreneurs
2. **Domain Expertise**: DeFi, NFTs, Games, Social, etc.
3. **Advanced Development**: https://hackmd.io/@nearly-learning/ncd-2

### TestNet Project Track

1. Build Up Teams
2. Course by Domain: DeFi / NFT / Game / …
3. Self-Paced Building & Releasing: Weekly Checkpoint (every Friday)
4. NEAR startup communities

#### Scope

1. Infrastructure & Tooling
    1. Infrastructure: Rainbow Bridge, Aurora, Octopus, NEAR Core
    2. SDK / API: NEAR Rust SDK, NEAR AS SDK, NEAR API JS
    3. Tools: NEAR CLI, NEAR Wallet, NEAR Explorer, Indexer Framework, etc.
2. Education: Documentation, Examples, Course, Learn NEAR Club, Videos (https://near-in-minutes.com/)
3. Applications: Ref Finance, Paras, Mintbase, Meta Pool

#### 中文社区贡献

**学习**

1. 中文开发者文档 / 学习资料
2. 协助 NCD 教学
3. 示例项目开发

**社区**

1. 发展 NEAR 中文社区
2. 连接其他开发者/开源社区

**工具**

1. 钱包
2. 浏览器

**项目**

1. [Bounties](https://near.org/bounties)
2. [Grants](https://near.org/grants/)

## Reference

1. Gitcoin Kernel: https://gitcoin.co/blog/announcing-kernel/

**Comment by think-in-universe** [comment]
**Week 1 Assignment (before Sep 24th): Introduce Yourself and Submit Proposal** 

Please introduce yourself, and submit your NCD L2 Proposal in this thread 

Please share your name, skill set, your team information, and the projects / tasks you'd like to achieve in NCD L2.

e.g. 示例（可以写中文）

### robertyan.near | Contributor for NEAR Protocol

Name: Robert Yan

Team: robertyan.near

Skills: Rust, C++, JavaScript, Python, Java, Scala / Full Stack, Data Science, Blockchain

Projects (details in the issue: https://github.com/near-x/learn-near-community/issues/34)

1. Wallet SDK standard: set up the wallet SDK standard, and release the 1st version of browser and mobile wallets
2. near-cookbook: complete the ebook that introduces the best practices for different domains
3. Launch NCD L2: launch and complete the 1st cohort of NCD L2

**Reply by lk2684753** [reply]
lk2684753.near | TestNet Project Track
Name: liangkai

Team: lk2684753.near jiahong.near zcsl_zoe.near

Skills: Go,Rust,C++,Blockchain,Distributed storage

Projects (details in the issue: #36)

We wanted to make a decentralized game;具体请参考 (details in the issue: #36)

**Reply by MagicGordon** [reply]
**bim.near | Contributor for Ref-finance**

Name: Guodong Zhang

Team: bim.near

Skills: Rust、C/C++、Java、Scala、Golang、Python、Flink(real-time computing)、intel SGX(trusted computing)

Projects (details in the issue: #41 )：

-  Create fuzzy test for the ref-exchange contract.
